recipes/pymonetdb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymonetdb" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "77e3bacd4a947bd54419939cdc4b4dcdee579edaad3a898c53c11b30695f3be9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - pymonetdb
+    - pymonetdb.control
+
+about:
+  home: http://www.monetdb.org
```

Comment:
It's not clear if this python client is officially supported by monetdb, so I'm not sure if this should be the home of this package. If it isn't, I would just make this either the github url or the docs.

Reply:
Seems this is the official python client:
https://github.com/MonetDB/MonetDB/blob/31d3c91730c94edd212a932ffb75b8cdad40f767/debian/changelog#L584-L586
http://pymonetdb.readthedocs.io/en/latest/introduction.html#introduction
https://www.monetdb.org/Documentation/Manuals/SQLreference/Programming/Python